app/views/mailers_common/_booking_confirmation.html.haml

```diff
@@ -114,3 +114,6 @@
 
 %p
   %small Visit ID: #{@visit.visit_id}
+
+%script{type: 'application/ld+json'}
```

Comment:
We should do this in HTML if we can, I think. It makes our markup more semantic and means the metadata is then a first-class citizen and less likely to be subject to the sort of scrutiny that JavaScript is (by less modern mail clients, anyway), as @mat mentions.